leetcode/src/5.c

```diff
@@ -0,0 +1,50 @@
+/**
+ * Find longest palindrome by traversing the string and checking how
+ * long palindrome can be constructed from each element by going left and right.
+ * Checking palindromes of types '..aa..' and '..bab..'
+ */
+
+#include <stdlib.h>
+#include <string.h>
```

Comment:
Not sure if we should include headers here. Usually, LeetCode problems do not provide these. 🤔

Reply:
I think it is quite hard to allocate string memory dynamically without including any libraries. My initial commit was without includes because I reused the provided string for returned result. By the way, leetcode webpage allows including the libraries for C solutions.